### DISCUSSION TITLE
如何解决每次接口调用uuid变化问题？

### DISCUSSION THREAD

**Discussion by hl10105210** [arextest/arex-agent-java#590] [General]
添加类接口，java后台会为添加的对象生成一个唯一id，采用的是uuid，每次请求会产生不一样的id返回，如何解决这类问题？

**Comment by lucas-myx** [comment]
建议先完整阅读下操作文档 [动态类配置](https://doc.arextest.com/zh-Hans/docs/chapter3/record-replay-config#%E5%8A%A8%E6%80%81%E7%B1%BB%E9%85%8D%E7%BD%AE)
